contrib/babelfishpg_common/src/typecode.c

```diff
@@ -300,6 +300,21 @@ lookup_tsql_datatype_oid(const char *typename)
 	return typoid;
 }
 
+/* type_infos will return const char * so called, should not attempt to modify it */
+void
+resolve_pg_type_to_tsql(Oid oid, char **typ_name)
```

Comment:
Why cant we return const char * instead of using a double pointer?
Seems inefficient for a caller to make sure its const

Reply:
It was crashing when return type was char*, crashing in https://github.com/amazon-aurora/babelfish_extensions/blob/57c3f503362b450b943610e68b09696cfdade278/contrib/babelfishpg_common/src/varchar.c#L104

Reply:
why would it crash if we return const char *?

Reply:
+1, not clear on why its crashing

Reply:
Made that change, crash was because earlier it wasn't defined as `Const`, worked now